examples/model_load_store_lora/service.py

```diff
@@ -0,0 +1,63 @@
+import os
+import typing as t
+
+from diffusers import StableDiffusionImg2ImgPipeline
+from diffusers import StableDiffusionPipeline
```

Comment:
let's put diffusers import inside service class's `__init__` function, that way it will lazily import the external library module, which may speedup build/start up time